### DISCUSSION TITLE
why a word  even not exist in spacy should have vector?

### DISCUSSION THREAD

**Discussion by Oscarjia** [explosion/spaCy#12536] [Help: Other Questions] [answered]
## How to reproduce the behaviour

i want to use word's vector to check if word is existed, but i found even not existed word in spacy model will have vector.

```
import spacy

# Load the English language model in spacy
nlp = spacy.load("en_core_web_md")

# Get a token for the word "apple"
word_token = nlp("becaue")[0]

# Check if the word token has a vector
if word_token.has_vector:
    print(f"The word '{word_token.text}' has a vector representation.")
else:
    print(f"The word '{word_token.text}' does not have a vector representation.")
```
the word `becaue` has vector, but `verson` don't have vector. So, why this will happen?

## Your Environment
* spaCy Version Used:3.5.1
* en_core_web_md:3.5.0

**Comment by adrianeboyd** [comment, marked as answer]
The words with vectors are words above a certain frequency in a corpus of primarily webcrawl data, so if a misspelling or typo is frequent enough, it may have a vector.

The list of words with vectors is large and noisy and probably isn't the right kind of word list to use for this task.
